### PR TITLE
Enable opensuse

### PR DESCRIPTION
**Pull request by clsulliv**
@thkukuk, could you give this a try?

Do be warned on the destructing nature of the configuration portions.

Thanks!

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -98,6 +101,12 @@ while getopts "a:d:hH:r:s:t:u" arg; do
           NGINX_USER="nginx"
           NGINX_GROUP="nginx"
           ;;
+        opensuse|suse)
+          DISTRO="opensuse"
+          NGINX_USER="nginx"
+          NGINX_GROUP="nginx"
+          REDIS_SERVICE="redis@default"
```
Comment (thkukuk) [inline scripts/deploy.sh]: To allow parallel installation of different redis services, I would propose to use "redis@telemetrics" here.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -418,6 +448,15 @@ _config_uwsgi_centos() {
   sudo semodule -i nginx-uwsgi.pp
 }
 
+_config_uwsgi_opensuse() {
+  sudo cp -afv $scripts_path/uwsgi.service /etc/systemd/system/
+  sudo systemctl daemon-reload
+}
+
+_config_redis_opensuse() {
+  sudo cp -afv /etc/redis/default.conf.example /etc/redis/default.conf
```
Comment (thkukuk) [inline scripts/deploy.sh]: To allow parallel installation of different redis services, I would propose to use "/etc/redis/telemetrics.conf" here.

**Comment by thkukuk** [comment]
I would use /etc/redis/telemetrics.conf and redis@telemetrics to avoid conflict with possible other services.
Else it's working fine for me on openSUSE Tumbleweed

**Comment by thkukuk** [comment]
Is this waiting for more input from me?

**Comment by clsulliv** [comment]
Apologies. No and thank you for your feedback. I got caught up in some other work and this got sidelined. I'll make changes based on your feedback and rebase against master today.

**Comment by thkukuk** [comment]
This looks good to me.

**Comment by clsulliv** [comment]
@alexjch could you look into merging this? I don't think it should do any harm. 

Commit 05a4cee9e may impact filtering on non-Clear Linux systems, though.

**Pull request merged**